pkg/cluster/sync.go

```diff
@@ -410,6 +421,52 @@ func (c *Cluster) syncStatefulSet() error {
 	return nil
 }
 
+func (c *Cluster) restartInstances() error {
+	c.setProcessName("starting to recreate pods")
```

Comment:
I assume here and elsewhere within this function what is meant here are Postgres. Please fix relevant log lines